### PR TITLE
Remove AuditEventService from SecurityConfig for reactive and no databaseType

### PR DESCRIPTION
**Pull request by mshima**
Fixes https://github.com/jhipster/generator-jhipster/issues/11620

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: .github/workflows/webflux.yml [hunk 1]
```diff
@@ -53,6 +53,7 @@ jobs:
                     - webflux-couchbase
                     - webflux-couchbase-es-oauth2
                     - webflux-psql
+                    - webflux-nodb-oauth2
```
Comment (murdos) [inline .github/workflows/webflux.yml:56]: We already have a lot of integration tests, we can't test all combinations on each PR and merge.
I don't think this one deserves to be tested (even for daily-builds).

Comment (mshima) [reply .github/workflows/webflux.yml:56]: I've put here to fix https://github.com/jhipster/generator-jhipster/issues/11620.

> I don't think this one deserves to be tested (even for daily-builds).

If so we should forbid reactive + 'no' databaseType.
CC @mraible

Comment (murdos) [reply .github/workflows/webflux.yml:56]: No, we don't need to forbid it. 
But we can't automatically test all possible combinations of all the options we provide, that would be too much. 
And this is why a few combination are tested here, and a few others there: https://github.com/hipster-labs/jhipster-daily-builds

Comment (mshima) [reply .github/workflows/webflux.yml:56]: Yep, I was planning to put the test there.
I just think fixing without adding a test to daily build seems wrong.

Comment (mshima) [reply .github/workflows/webflux.yml]: Removed.

**Comment by mshima** [comment]
@mraible I couldn't get the e2e test to succeed.
I fails because the admin user doesn't exists.
I have no clue how to fix it.

**Comment by mraible** [comment]
@mshima It looks like all tests are passing. Do you have a link to the failing e2e test?

**Comment by mshima** [comment]
@mraible e2e fail is here: https://github.com/jhipster/generator-jhipster/runs/613106673

**Comment by mraible** [comment]
Thanks. I’ll take a look in the next 24 hours.

**Comment by mraible** [comment]
@mshima I believe the issue is caused by the new `onAuthenticationSuccess()` handler. Previously, it was:

```java
private Mono<Void> onAuthenticationSuccess(WebFilterExchange exchange, Authentication authentication) {
    return redirectServerAuthenticationSuccessHandler.onAuthenticationSuccess(exchange, authentication)
        .thenReturn(authentication.getPrincipal())
        .filter(principal -> principal instanceof OidcUser)
        .map(principal -> ((OidcUser) principal).getPreferredUsername())
        .filter(login -> !Constants.ANONYMOUS_USER.equals(login))
        .flatMap(auditEventService::saveAuthenticationSuccess)
        .then();
}
```

Now it's:

```java
private Mono<Void> onAuthenticationSuccess(WebFilterExchange exchange, Authentication authentication) {
    return Mono.empty();
}
```

If you run the app, you'll see that you get a blank page after you log in to Keycloak. I'd get rid of the `onAuthenticationSuccess()` and `redirectServerAuthenticationSuccessHandler` variable altogether for oauth2 with no DB. I tried this locally and e2e tests pass.

**Comment by mraible** [comment]
@mshima I pushed some changes that should fix e2e tests.

**Comment by mshima** [comment]
Thanks @mraible, I am focused on https://github.com/jhipster/generator-jhipster/pull/11423 right now.

**Comment by mraible** [comment]
Tests are failing because of an upgrade to couchmove. https://github.com/jhipster/jhipster/pull/679 should fix the problem, once it's merged.

**Comment by kamilj** [comment]
How can I install version 6.9.0? It is not available as master in npm packages.

**Comment by mraible** [comment]
@kamilj You can't. It hasn't been released yet! 😊

If you want to try things out, you can use the `master` branch.

```
git clone git@github.com:jhipster/generator-jhipster.git
cd generator-jhipster
npm link
```

Then generate your app and re-link generator-jhipster.

```
mkdir app && cd app
jhipster
npm link generator-jhipster
```

**Comment by kamilj** [comment]
~/generator-jhipster-master# npm link

> ejs@3.0.1 postinstall /root/generator-jhipster-master/node_modules/ejs-lint/node_modules/ejs
> node ./postinstall.js

internal/modules/cjs/loader.js:960
  throw err;
  ^

Error: Cannot find module '/root/generator-jhipster-master/node_modules/ejs-lint/node_modules/ejs/postinstall.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
    at Function.Module._load (internal/modules/cjs/loader.js:840:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ejs@3.0.1 postinstall: `node ./postinstall.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ejs@3.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-05-14T19_40_16_528Z-debug.log

**Pull request merged**